CHANGELOG.txt

```diff
@@ -1,5 +1,11 @@
-4.0.2
+4.1.0
+* Exposes billing metrics
+* Adds runtime environment headers for http fauna requests
+* Always sends query timeout header
+* Enables user to pass a custom user agent
 * Adds ability to combine Fields
+* FaunaClient no longer needs to be closed
+* defect fixes
```

Comment:
Can we be more precise here? What does it mean?

Reply:
I've added one more commit to the list:
- Expr(NullV) is instantiated only once and reused
This was the only one I missed, I removed "defect fixes" from changelog